AWESOME.md

```diff
@@ -0,0 +1,35 @@
+Awesome AutoGluon
+-----------------
+
+This page contains a moderated list of examples, tutorials, articles, and research papers about AutoGluon use cases.
+It is inspired by [awesome-machine-learning](https://github.com/dmlc/mxnet/blob/master/example/README.md).
+
+We will be happy to add your success story using AutoGluon to this list.
+Send us a pull request if you want to include your case here.
+
+## Videos & Tutorials
+* [AutoML Fall School 2022 - Hands-on tutorial AutoGluon](https://www.youtube.com/watch?v=VAAITEds-28)
+  * This is the most up-to-date indepth tutorial of AutoGluon, presented by @innixma
+
+## Competition Solutions using AutoGluon
+
+### Kaggle
+
+#### Tabular
+
+1. (Live, 2023/01/09) Rank 1/689 : [Tabular Regression with the California Housing Dataset](https://www.kaggle.com/competitions/playground-series-s3e1/discussion/377137)
+2. (Live, 2023/02/20) Rank 9/703 : [Tabular Regression with a Paris Housing Price Dataset](https://www.kaggle.com/competitions/playground-series-s3e6/discussion/389151)
+3. (Live, 2023/03/06) Rank 6/734 : [Tabular Regression with a Gemstone Price Dataset](https://www.kaggle.com/competitions/playground-series-s3e8/discussion/392820)
+4. (Live, 2023/03/08) Rank 2/58 : [ML Olympiad - Water Quality Prediction](https://www.kaggle.com/competitions/ml-olympiad-waterqualityprediction/discussion/393393)
+5. (Live, 2023/03/11) Top 3% : [Titanic - Machine learning from Disaster](https://www.kaggle.com/competitions/titanic/discussion/394105#2178151)
+6. (Hist,  2022/03/22) Rank 24/3507 : [Otto Group Product Classification Challenge](https://www.kaggle.com/code/innixma/autogluon-rank-24-in-1-line-of-code)
+
+## AutoML Benchmarks using AutoGluon
+* [AMLB: An AutoML Benchmark](https://openml.github.io/automlbenchmark/)
+  * For a thorough comparison of AutoGluon and other modern AutoML systems, please refer to the 2022 paper ["AMLB: An AutoML Benchmark"](https://arxiv.org/abs/2207.12560) where AutoGluon is shown to be the state-of-the-art among AutoML systems on tabular data.
+  * We encourage all users to benchmark AutoGluon & other AutoML frameworks on AMLB.
+  * This is our preferred benchmark as it is widely accepted and trusted within the AutoML community.
+
+## Papers using AutoGluon
```

Comment:
Shall we have a section for *Tools with AutoGluon* which can include cloud and benchmark? Also community would be helpful as well (e.g. Discord, Slack)